packages/access-decision-manager/src/index.ts

```diff
@@ -1,4 +1,7 @@
 import AccessDecisionManager from './access-decision-manager';
 
 export * from './voter';
+export * from './strategy'
+export * from './strategy/'
```

Comment:
This line is duplicated!

Reply:
One export is for the `strategy.ts` file, which contains the `Strategy` type, and the other for the strategy directory from where we export the `unanimous` and `affirmative` strategies

Reply:
Also I wondering what type of commit to use, im thinking of `refactor` but also `feat`, what do you think Leo?

Reply:
I'd use `chore:`